src/OpenTelemetry.Instrumentation.AspNetCore/OpenTelemetry.Instrumentation.AspNetCore.csproj

```diff
@@ -5,6 +5,10 @@
     <PackageTags>$(PackageTags);distributed-tracing;AspNetCore</PackageTags>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="..\OpenTelemetry.Instrumentation.Dependencies\GrpcTagHelper.cs" />
```

Comment:
I did this as a quick way to get something working, though this definitely feels goofy to me. It didn't make sense to add a dependency from this project to the OpenTelemetry.Instrumentation.Dependencies project.

The `GrpcTagHelper` class is shared between the client and server instrumentation.

Few options:
* Is there a better project for this class to belong?
* Maybe the gRPC functionality for ASP.NET Core should be separate from the AspNetCore instrumentation? That is, the Grpc.AspNetCore instrumentation could be moved to the Dependencies project. Perhaps creating a separate `HttpInListener` just for the gRPC functionality.

Reply:
@alanwest , what i have seen so far: in OpenTelemetry/Internal, we have some clases that are shared the same way you did. If you check the class `DateTimeOffsetExtensions.net452.cs`, I added in almost all classes that we needed to target to NET452. Don't know exactly if thats a good/bad practice

Reply:
@eddynaka Yea, I think moving `GrpcTagHelpers` to OpenTelemetry/Internal is a decent idea. You think it makes better sense there than leaving it in the Instrumentation.Dependencies project?

Reply:
@alanwest , i would wait others to comment before change everywhere. But, yes, at least, you won't have references everywhere causing a "cycle" reference...